.idea/gradle.xml

```diff
@@ -4,7 +4,7 @@
   <component name="GradleSettings">
     <option name="linkedExternalProjectsSettings">
       <GradleProjectSettings>
-        <option name="testRunner" value="PLATFORM" />
+        <option name="testRunner" value="GRADLE" />
```

Comment:
All the changes made in .idea/gradle.xml files are system generated. Please don't push these changes.